pkg/client/generate_models.go

```diff
@@ -226,6 +226,23 @@ var groupModel = model{
 	},
 }
 
+var statisticsModel = model{
+	name: "statistics",
+	fields: []modelField{
+		{name: "documents_total", typ: "int64", readOnly: true},
+		{name: "documents_inbox", typ: "int64", readOnly: true},
+		{name: "inbox_tag", typ: "int64", readOnly: true},
+		{name: "inbox_tags", typ: "[]int64", readOnly: true},
+		{name: "document_file_type_counts", typ: "[]DocumentFileType", readOnly: true},
+		{name: "character_count", typ: "int64", readOnly: true},
+		{name: "tag_count", typ: "int64", readOnly: true},
+		{name: "correspondent_count", typ: "int64", readOnly: true},
+		{name: "document_type_count", typ: "int64", readOnly: true},
+		{name: "storage_path_count", typ: "int64", readOnly: true},
+		{name: "current_asn", typ: "int64", readOnly: true},
+	},
+}
+
```

Comment:
Can you please switch to a hand-written struct to not need `readOnly` on every member? Like in #42.

Reply:
Done via https://github.com/hansmi/paperhooks/pull/43/commits/31add1bea84b73be972d8dc886ab58564a7d51b8